pkg/agent/openflow/pipeline.go

```diff
@@ -2410,9 +2407,33 @@ func (f *featureService) serviceLBFlow(groupID binding.GroupIDType,
 	if f.enableAntreaPolicy {
 		flowBuilder = flowBuilder.Action().LoadToRegField(ServiceGroupIDField, uint32(groupID))
 	}
+	if nested {
+		flowBuilder = flowBuilder.Action().LoadRegMark(NestedServiceRegMark)
+	}
```

Comment:
Maybe add some comments here. This reg mark is loaded by MC Service, not the normal Service. I thought of it is loaded by normal Service since the normal Service is nested in MC Service.

Reply:
Done